cache/in-memory/src/model/emoji.rs

```diff
@@ -4,29 +4,41 @@ use twilight_model::{
     id::{EmojiId, RoleId, UserId},
 };
 
+/// Represents a cached [`Emoji`].
+///
+/// [`Emoji`]: twilight_model::guild::Emoji
 #[derive(Clone, Debug, Eq, PartialEq, Serialize)]
 pub struct CachedEmoji {
-    pub id: EmojiId,
```

Comment:
Maybe we should order these under a separate PR to keep the concerns of this PR limited to the topic?

Reply:
I do like the change though